build.sh

```diff
@@ -123,6 +123,17 @@ set -x # Print commands from now on
     --disable-lzo \
     --disable-smartcard \
     --disable-usb-redir \
+    --disable-bochs \
+    --disable-cloop \
+    --disable-dmg \
+    --disable-vdi \
+    --disable-vvfat \
+    --disable-qcow1 \
+    --disable-qed \
+    --disable-parallels \
+    --disable-sheepdog \
+    --without-default-devices \
```

Comment:
I just noticed that I didn't try testing any uncommon devices like usb-forwarding or usb-storage.
Could this be excluded by this option? We should check that.

Reply:
This is actually a problem:

qemu-system-i386: -device usb-storage,drive=xmu_1a_path,port=3.2: 'usb-storage' is not a valid device model name

This feature is exposed from XQEMU-Manager, so this would break it.

Reply:
Using a Duke through the `usb-host` driver works like before.

Reply:
Good catch. Will fix!

Reply:
Fixed. usb-storage is enabled again. Requesting someone help test